### PR TITLE
Hide Data Quality by making it a draft

### PR DESCRIPTION
**Pull request by alexanderahn**

### CODE REVIEW

File: docs/release_notes/2023/apr2023.md [hunk 1]
```diff
@@ -41,7 +41,6 @@ Data quality refers to the accuracy, relevance, completeness, and consistency of
 There are several factors that affect data quality. These include data entry errors, data duplication, missing data, inconsistent data, and outdated data. These issues can arise from human error, system errors, or technical issues such as data integration problems.
 
 In Prophecy, you, as a user, can now easily set **Expectations** on your data such as uniqueness, range, and more. You can then schedule these **Expectations** to run at fixed **Schedules**, and receive alerts whenever any mismatch occurs, empowering you to maintain the quality of your data.
```
Comment (atbida) [inline docs/release_notes/2023/apr2023.md:43]: I suggest removing this section #### Data Quality (_Beta_)

File: docs/release_notes/2023/apr2023.md [hunk 2]
```diff
@@ -72,7 +71,6 @@ Please see below screenshot for example
 #### New Data Observability Page
 
 Prophecy has updated the UI for the Data Observability page (previously known as Monitoring). You can now monitor the Data quality runs as well as Jobs from this page.
```
[the remark this answers is not in the document]
Comment (alexanderahn) [reply docs/release_notes/2023/apr2023.md:73]: Done.

File: docs/release_notes/2023/apr2023.md (hunk 1 above)
Comment (alexanderahn) [reply docs/release_notes/2023/apr2023.md:43]: I was hesitant on completely removing items from published release notes, but I'll go ahead with your suggestion.

**Review by atbida — APPROVED** [review]
lgtm!

**Pull request merged**